test/topojson/test.ml

```diff
@@ -55,12 +55,23 @@ let expected_arcs =
       |];
     |]
 
+let expected_foreign_members =
```

Comment:
This is not required, the expected foreign members is a JSON value not a function